### DISCUSSION TITLE
Welcome to clonenetflix Discussions!

### DISCUSSION THREAD

**Discussion by iamshakkib** [iamshakkib/clonenetflix#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Roy-droid-2496** [comment]
Hi, I cloned your project into my local machine, but am not able to run it , can you please help me out.

**Reply by iamshakkib** [reply]
Hi @Roy-droid-2496, thank you very much for following along and also for waiting patiently. May you post the error you're getting in the logs from IntelliJ or any other IDE ? And is your backend server and frontend server up in your local machine
